### DISCUSSION TITLE
Hold Joystick to Scroll thru long Menus

### DISCUSSION THREAD

**Discussion by bryanward-net** [WLAN-Pi/feedback#46] [Feature Request]
It would be nice to be able to hold the joystick up/down to automatically scroll thru long menus (such as timezone).

**Comment by nickjvturner** [comment]
As of WLPC_PHX_2023, we introduced fpms looping, when you reach the end of a list the next downward navigation will take you back up to the top, the same works in reverse. Does this resolve your specific use-case, or do you still view 'hold-navigation scroll' to be be essential / desirable.

**Comment by bryanward-net** [comment]
Essential, no.  Desirable, yes.
I put this FR in when setting the timezone of my Pi and was having to click-click-click thru the whole list.
Aside from that, I haven't come across any other scenario where I thought this was needed.
Perhaps for future uses of the screen - if some app were to list scanned SSIDs or do neighbor discovery - where a long list might be present then I think this would be a good feature.
